setup-tdx-guest.sh

```diff
@@ -7,6 +7,8 @@ if [ ! -z "${TDX_GUEST_SETUP_INTEL_KERNEL}" ]; then
    KERNEL_RELEASE=6.8.0-1001-intel
 fi
 
+source ${SCRIPT_DIR}/setup-tdx-common
```

Comment:
Since this is only bringing in `add_kobuk_ppa`, perhaps we should move it down in the script immediately above that function for better clarity. What do you think?

Reply:
this script has only 1 function for now but it is intended to have more in the future, so i think we can let it here because if we add a new function, we will have to move it again on top of the file

Reply:
Sounds good :+1: